libs/zzz/ui/src/Disc/DiscSetCardCompact.tsx

```
@@ -0,0 +1,87 @@
+          {Object.entries(sets).map(([key, count]) => (
+            <Box
+              component="div"
+              sx={{
+                display: 'flex',
+                justifyContent: 'space-between',
+                background: '#2B364D',
+                py: '16px',
+                px: '5px',
+                borderRadius: '12px',
+              }}
+            >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)